app/styles/ui/_tab-bar.scss

```diff
@@ -3,6 +3,10 @@
   flex-direction: row;
   height: var(--tab-bar-height);
 
+  .tab-container {
+    overflow: hidden;
+  }
```

Comment:
The reason for this is that, so far, we're giving fixed widths to our all the dialogs (`Preferences` and `Repository Settings`) where a tab-bar is used, and in any case it doesn't make sense to allow them to grow beyond the boundaries of their parents.

What do you think?

Reply:
Actually, I tested again without this change and seems to work just fine, so I removed it. Nothing to see here… 👀